plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx

```diff
@@ -1,16 +1,18 @@
 import { action, makeObservable, observable, ObservableMap } from 'mobx';
-import { registerHandler, unregisterHandler, WSMsgType } from 'WS';
+import { registerHandler, unregisterHandler, WSMsgType } from '../WS';
```

Comment:
The baseUrl is set to `src` in tsconfig.json, `WS` should work. :thinking:

Reply:
For some reason `ws` was conflicting with some node_modules stuff. I think we should leave it as `../ws` to be safe.